pkgs/development/python-modules/jupyterhub-ldapauthenticator/default.nix

```diff
@@ -0,0 +1,32 @@
+
+{ lib
+, buildPythonPackage
+, jupyterhub
+, ldap3
+, pythonOlder
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "jupyterhub-ldapauthenticator";
+  version = "1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "19dz3a3122wln8lkixj5jbh9x3cqlrcb3p7a53825cj72cmpcxwz";
+  };
+
+  # No tests in archive
```

Comment:
There aren't even any on GitHub, so this should say something like `No tests implemented`